requirements/rtd.txt

```diff
@@ -1,4 +1,8 @@
 
--r docs.txt
```

Comment:
we still need to include `-r base.txt` and `-r dev.txt` as well.

Reply:
GRUMBLE GRUMBLE